model/__init__.py

```diff
@@ -782,3 +782,7 @@ def set_certificate(self, certificate: str):
 
     def _required_attributes(self):
         return ['private_key', 'certificate']
+
+class SlackConfig(NamedModelElement):
+    def api_token(self):
+        return self.raw.get('api_token')
```

Comment:
as it does not make sense to have this configuration element without an api_token, implement a method _require_attributes and return api_token